### ISSUE TITLE
Entities in validation sets

### ISSUE THREAD

**Issue by chjjma** [nju-websoft/MBE#2]
Hi. 

First, thanks for the code.
In the dataset section of the paper, one of the constraints for cleaning is: the set of entities of the validation set is a subset of the set of entities of the training set.
However, for the datasets released, it seems that this constraint is not true.
In WN-MBE, entity '1828267' is in the validation set, but not in the training set.
In NELL-MBE, entity 'concept_athlete_jon_kitna' is in the validation set, but not in the training set.
In FB-MBE, entity '/m/01pf6' is in the validation set, but not in the training set.
It seems that the statistics provided in the paper is before cleaning.
Could you provide the statistics after cleaning?
Thanks in advance.

**Comment by yncui-nju** [comment]
Hi,

Thanks for your interest in our work! We have updated the dataset. There are 7636, 31406, and 18501 validation facts in WN-MBE, FB-MBE, and NELL-MBE, respectively.

**Comment by chjjma** [comment]
Thanks for the update!
But I think there are still some entities not in the train set but in the entity set.
For example, /m/05_p2 in FB-MBE's valid.triples but not in base_train.triples
Thanks in advance.

**Comment by yncui-nju** [comment]
Thanks! These entities have been deleted.

**Comment by chjjma** [comment]
Thank you!
